client/blocks/inline-help/admin-sections.js

```diff
@@ -0,0 +1,381 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { memoize } from 'lodash';
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal Dependencies
+ */
+
+/**
+ * Returns admin section items with site-based urls.
+ *
+ * @param {String} siteSlug - The current site slug.
+ * @returns {Array}           An array of admin sections with site-specific URLs.
+ */
+export const adminSections = memoize( ( siteSlug ) => [
+	{
+		title: translate( 'Add a new domain' ),
+		description: translate(
+			'Set up your domain whether it’s registered with WordPress.com or elsewhere.'
+		),
+		link: `/domains/add/${ siteSlug }`,
+		synonyms: [ 'domain' ],
+		icon: 'domains',
+	},
+	{
+		title: translate( 'Manage my domain settings' ),
+		link: `/domains/manage/${ siteSlug }`,
+		icon: 'domains',
+	},
+	{
+		title: translate( 'Change my site address' ),
+		link: `/domains/manage/${ siteSlug }/edit/${ siteSlug }`,
+		synonyms: [ 'domain' ],
+		icon: 'domains',
+	},
+	{
+		title: translate( 'Add a site redirect' ),
+		description: translate( 'Redirect your site to another domain.' ),
+		link: `/domains/add/site-redirect/${ siteSlug }`,
+		synonyms: [ 'domain', 'forward' ],
+		icon: 'domains',
+	},
+	{
+		title: translate( 'Change my password' ),
+		link: '/me/security',
+		synonyms: [ 'update' ],
+		icon: 'cog',
+	},
+	{
+		title: translate( "Change my site's theme" ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Change your site's address' )` *ES Score: 7*